modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationHistoryPropsTransformer.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayProgressBar from '@clayui/progress-bar';
+import {createPortletURL, fetch} from 'frontend-js-web';
+import React, {useEffect, useState} from 'react';
+
+export default function propsTransformer({
+	additionalProps: {getPublicationStatusURL},
+	...otherProps
+}) {
+	const PublicationHistoryStatusRenderer = (props) => {
```

Comment:
Let's do an early return if status is not in-progress. Only add the progress bar if `props.value.label` is `in-progress`. Otherwise, render the status label directly.